examples/model_selection/plot_multi_metric_evaluation.py

```diff
@@ -49,6 +49,8 @@
     param_grid={"min_samples_split": range(2, 403, 10)},
     scoring=scoring,
     refit="AUC",
+    cv=3,
```

Comment:
I don't think we want to reduce CV to 3, especially since people tend to copy/paste code.

Did you check the effect of reducing the number of samples?

Reply:
@adrinjalali Agreed, we want to keep it generic :)

Removed the ``cv`` param and decreased the sample number from 8000 to 6000: result is slightly worse but still 2X faster, so a good compromise I'd say
![mm_before](https://user-images.githubusercontent.com/66701689/141365918-660b2d5c-8f34-4754-a755-3206ed9ece73.png)
![mm_after](https://user-images.githubusercontent.com/66701689/141365929-49099fe6-b187-45ef-b773-c9994a0f84e6.png)